### PR TITLE
Home ui

### PR DESCRIPTION
**Pull request by vineet-webonise**

### CODE REVIEW

**Comment by tejasAlsi-Webonise** [comment]
@vineet-webonise please read the basecamp carefully, now its fine you added the moonkit but we may not need in future, as everything will come from backend.

**Comment by vineet-webonise** [comment]
> @vineet-webonise please read the basecamp carefully, now its fine you added the moonkit but we may not need in future, as everything will come from backend.

@tejasAlsi-Webonise  Got it, will look into and do the needfull. Thanks for knowing :)

File: SUNFLOWR/SUNFLOWR/HomePageViewController.swift [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+import UIKit
+
+class HomePageViewController: UIPageViewController {
+    
+    fileprivate var items: [UIViewController] = []
+    let stringArray = ["🎖", "👑", "🥇"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
```
Comment (saketPandhare-Webonise) [inline SUNFLOWR/SUNFLOWR/HomePageViewController.swift:14]: Add one line ofter super.viewDidLoad and remove blank space on line no 16

Comment (vineet-webonise) [reply SUNFLOWR/SUNFLOWR/HomePageViewController.swift:14]: Done

**Pull request merged**